docs/conf.py

```diff
@@ -0,0 +1,40 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+
+import os
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "LINCC Frameworks Python Project Template"
+copyright = "2023, LINCC Frameworks"
+author = "LINCC Frameworks"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+master_doc = "index"  # This assumes that sphinx-build is called from the root directory
+html_show_sourcelink = (
+    False  # Remove 'view source code' from top of page (for html, not python)
+)
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "alabaster"
```

Comment:
I (personally) would recommend not using 'alabaster' here. I just don't like it. Also, it's a bit strange to generate the docs locally using sphinx_rtd_theme, and have the actual documentation look different on readthedocs rendered with 'alabaster'. 

So I think that line 26 can be removed, and the conditional on line 34 and 35 can be as well.

Reply:
I copied this from the python project template, so maybe we should update it there too =]